controllers/articles-controllers.js

```diff
@@ -1,17 +1,48 @@
 const {
     selectArticleById,
+    selectArticles,
+    selectCommentsByArticle
     
   } = require('../models/articles-models.js');
   
   exports.getArticleById = (req, res, next) => {
-    console.log("fjdijsoijdifjdoijdi")
-    const id = req.params.article_id
-    console.log(id)
-    selectArticleById(id).then((article) => {
+    
+    const {article_id} = req.params
+    
+    
+    selectArticleById(article_id).then((article) => {
+      console.log(article, "Article")
       
+      if (article.length===0 && !isNaN(article_id)) {res.status(404).send({ msg:'article doesn\'t exist' });
```

Comment:
You should be finding errors in the model to pass back to your .catch(), see my comments below